code/modules/power/supermatter/supermatter_glow.dm

```diff
@@ -60,7 +70,7 @@
 			var/icon/ball = new/icon('icons/obj/engine/energy_ball.dmi', "energy_ball", frame = rand(1,12)) // Only 12 frames
 			var/tesla_filter = filter(type="layer", icon = ball, flags = FILTER_UNDERLAY)
 			our_supermatter.set_light(
-				initial(our_supermatter.light_range) + clamp(our_supermatter.damage*our_supermatter.internal_energy, 50, 500),
+				ROUND_UP(initial(our_supermatter.light_range) + clamp(our_supermatter.damage*our_supermatter.internal_energy, 10, 100)),
```

Comment:
100 light range is still WAY too high. 10 is borderline high.
100 light range would be 200x200 tiles, or 40000 tiles to update.
Clamp this between like 1 and 10 or so, it'll do what you want just fine